docs/1-tendermint-overview.md

```diff
@@ -0,0 +1,120 @@
+# Tendermint protocol
+
+We are going to present a short and simple overview of the [Tendermint](https://github.com/tendermint) consensus algorithm and the main aspects that are required to understand the fork accountability problem. 
+Please note that this is not an exhaustive and formal discussion. We invite the reader to check out the [Tendermint documentation](https://docs.tendermint.com/master/) and the [official paper](https://arxiv.org/abs/1807.04938) that describe the theoretical aspects in more detail.   
+
+## Consensus algorithm
+
+Tendermint is a partially synchronous Byzantine fault tolerant (BFT) consensus protocol. The protocol requires a fixed set of *n* validators that attempt to come to consensus on one value or, more precisely, on a block (a block is a list of transactions) using gossip-based communication. 
+
+Tendermint is notable for its simplicity and performance and is currently one of the most mature and widely used implementations of BFT consensus.
+
+### Properties 
+
+The Tendermint consensus algorithm guarantees the following properties, with respect to a consensus instance:
+
+- **Agreement**: No two correct processes decide on different values
+
+- **Validity**: A value decided is a valid value, with respect to the specifications of validity of a message
+
+- **Termination**: All correct processes eventually decide
+
+- **Integrity**: No correct process decides more than once in a single height 
+
+In order to ensure all these properties, the Tendermint consensus algorithm assumes a correct majority of processes such that less than one third of processes are faulty (*f*):
+ 
+    n > 3f
+ 
+When a process *p* is faulty, *p* can behave in an arbitrary way and we have no assumptions or guarantees regarding its actions.
+ 
+In order to simplify the whole discussion, we assume the following from now on:
+ 
+    n = 3f + 1
+
+### Overview
+
+The consensus algorithm proceeds in rounds: in each round there is a validator that proposes a value (proposer) and all the validators then vote during the round on whether to accept the proposed value or move on to the next round (proposers are chosen according to their voting power).
```

Comment:
I wouldn't introduce the voting power since we do not talk about it before. 
We need to be careful here.

Reply:
Ok I'll remove it